clans/src/main/java/me/mykindos/betterpvp/clans/progression/perks/TreeFeller.java

```diff
@@ -83,17 +84,23 @@ public void fellTree(Clan playerClan, Block block, PlayerChopLogEvent event, boo
         block.breakNaturally();
         event.setAmountChopped(event.getAmountChopped() + 1);
 
+        ArrayList<Integer> yValues = new ArrayList<>();
+        if (initialBlock) yValues.add(0);
+        yValues.add(1);
+
         for (int x = -1; x <= 1; x++) {
             for (int z = -1; z <= 1; z++) {
-                Block targetBlock = block.getRelative(x, 1, z);
+                for (int y : yValues) {
```

Comment:
for (int y = 0; y < 1; y++) would suffice, rather than making the list just to add an int to it

Reply:
scrapped that change. tbh it's not worth changing. But the cooldown msg was really bothering me so just that.